submodules/tags/main.tf

```diff
@@ -12,6 +12,9 @@ locals {
     "aws/account_number" = data.aws_caller_identity.current.account_id
```

Comment:
Please don't make me ask again you to format the code properly.

Reply:
Done.